From 39e03f574c34e89b8ee9aa07ea5a2f7a595ecc20 Mon Sep 17 00:00:00 2001
From: Marinus Schraal <mschraal@gnome.org>
Date: Tue, 8 May 2018 11:51:36 +0200
Subject: [PATCH] window: Set correct default icon name

With the switch to Meson the default icon name is now org.gnome.Music.
Fixes the icon not showing in gnome-control-center.


(cherry picked from commit eeea5f41b9a9f7e716f9b95dddfe2d151a06f06d)
---
 gnomemusic/window.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gnomemusic/window.py b/gnomemusic/window.py
index 956d7ab4..66131d72 100644
--- a/gnomemusic/window.py
+++ b/gnomemusic/window.py
@@ -77,7 +77,7 @@ class Window(Gtk.ApplicationWindow):
         self.add_action(select_none)
 
         self.set_size_request(200, 100)
-        self.set_default_icon_name('gnome-music')
+        self.set_default_icon_name('org.gnome.Music')
 
         self.prev_view = None
         self.curr_view = None
-- 
2.17.0

